### DISCUSSION TITLE
setconstant js not work

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#1991] [Q&A]
Firefox android 97.1.0
Ublock origin 1.41.4

URL= `https://akashort.com/ypweiss2`

filter `akashort.com##+js(set, blurred, false)` doesn't work on firefox android 97.1.0 but work on other desktop browsers 

i have tested couple of  `##+js(set, blurred, false)` in ublock filters, that also not works

**Comment by ghajini** [comment]
@gwarser

**Reply by ghost** [reply]
From what I see, the variable is correctly set, but this code does not run when in background.

Can be because intervals and timeouts are slowed-down or even suspended in background on mobile.

If I set breakpoints in both branches, switch to other tab and unpause, then nothing happens.

**Reply by ghajini** [reply]
is this a browser issue then?
if then i will try to report to firefox devs about their review

**Reply by gorhill** [reply]
`https://akashort.com/ypweiss2` = 404

**Reply by ghajini** [reply]
try this link instead

`https://apkshrt.com/hXgZ92y`

**Reply by ghajini** [reply]
so a filter such as `paperzonevn.com##+js(aeld, visibilitychange)` also not work on firefox android

filter is about to address https://github.com/uBlockOrigin/uAssets/issues/8053#issuecomment-995484869
